test/extended/util/cli.go

```diff
@@ -35,10 +37,19 @@ type CLI struct {
 	finalArgs       []string
 	stdout          io.Writer
 	verbose         bool
+	execBashArgs    bool
 	cmd             *cobra.Command
 	kubeFramework   *e2e.Framework
 }
 
+type ExitError struct {
+	ExitStatus int
+}
+
+func (e ExitError) Error() string {
+	return fmt.Sprintf("Program exited with status %d", e.ExitStatus)
```

Comment:
I felt it would be better for the user of the ExitError to be able to extract the value, in case he'd want to ignore a particular error.